dbt/adapters/glue/connections.py

```diff
@@ -42,11 +41,6 @@ def open(cls, connection):
             connection.state = GlueSessionState.FAIL
             raise FailedToConnectException(f"Got an error when attempting to open a GlueSessions: {e}")
 
-    def cancel(self, connection):
-        """ cancel ongoing queries """
-        logger.debug("Cancelling queries")
-        connection.handle.cancel()
-        logger.debug("Queries canceled")
```

Comment:
do you need all these `log` statments still? dbt-core and dbt-spark _should_ handle this logging